packages/components/app/styles/components/alert.scss

```diff
@@ -0,0 +1,31 @@
+//
+// ALERT
+//
+// properties within each class are sorted alphabetically
+//
+//
+
+.hds-alert {
+  background-color: var(--token-color-surface-warning);
+  padding: 1.25rem; // 20px
+  display: flex;
+  align-content: flex-start;
+}
+
+.hds-alert__icon {
+  margin-right: 0.75rem; // 12px
+}
+
+.hds-alert__text {
+  color: var(--token-color-foreground-warning-on-surface);
+  font-family: var(--token-typography-body-200-font-family);
+}
+
+.hds-alert__title {
+  font-weight: var(--token-typography-font-weight-semibold);
+  margin: 0; // TODO: Temp override, p has margin, why?
+}
+
+.hds-alert__body {
```

Comment:
instead of doing this (and when you have just the `body/description` you have an extra margin top) you can do this:
```
.hds-alert__title + .hds-alert__body {
    margin-top: 0.75rem;
}
```

Reply:
i get what you mean, there's some margin on every p on the scrappy site. so in this instance, i'm overriding individually (the overrides shouldn't be in the component code at all). will address the p separately though

i added it as a TODO and will fix it upstream https://github.com/hashicorp/design-system/pull/92/commits/e86193fa719e8c80176700d3fd8eee625e682946

Reply:
@amyrlam the margin on the `<p>` elements is the default one, provided by the user agent stylesheet:
<img width="354" alt="screenshot_1128" src="https://user-images.githubusercontent.com/686239/159234427-c490ec29-f42b-4fd7-8f4d-56f4c8ac2b26.png">
we don't have global resets, that's why one needs to set the margin/padding on elements like `<p>` and `<li>` when used inside components

Reply:
@didoo gotcha, I ended up deciding to remove it anyways, thinking of all the globals in consuming apps